launcher/InstanceCopyPrefs.h

```diff
@@ -0,0 +1,20 @@
+//
```

Comment:
Do we need a separate .h/.cpp file for this? maybe we could just have it inside `InstanceCopyTask`

Reply:
Okay, I'll work on that!

Reply:
Although, if I put it in InstanceCopyTask then we'll have to include InstanceCopyTask in CopyInstanceDialog even though we'll only use that struct and it won't be instantly clear where the struct comes from, does that matter?

Reply:
huh, i thought it already included it.
no need to change this then :P